infra/terraform/modules/data_access/team_events/requirements.dev.txt

```diff
@@ -0,0 +1,5 @@
+boto3
+
+pytest
+pytest-spec
+pytest-watch
```

Comment:
Do we need a requirements file for each directory of lambda functions? In dev, we could just have one virtualenv for all of them, couldn't we? Since there's only `requirements.dev.txt`s, I'm assuming there's no need for a requirements file in production, right? The lambda python environment has boto3 preloaded?